daphne_worker/src/router/leader.rs

```diff
@@ -97,7 +97,7 @@ pub(super) fn add_leader_routes(router: DapRouter<'_>) -> DapRouter<'_> {
                     Ok(DapCollectJob::Done(collect_resp)) => dap_response_to_worker(DapResponse {
                         version: DapVersion::Draft02,
                         media_type: DapMediaType::Collection,
-                        payload: collect_resp.get_encoded_with_param(&version),
+                        payload: collect_resp.get_encoded_with_param(&version).expect("XXX"),
```

Comment:
@mendess how should we handle this encoding failure? Previously we would have panicked.

Reply:
See previous suggestion on the other routing method. But here use `daph.state.dap_abort_to_worker_response(e)`